guides/common/modules/ref_registration-methods.adoc

```diff
@@ -0,0 +1,26 @@
+[id="registration-methods_{context}"]
+= Registration Methods
+
+There are the following main methods for registering hosts:
+
+Global registration::
++
+You generate a `curl` command from {Project} and run this command from an unlimited number of hosts to register them using the global registration template.
```

Comment:
```suggestion
Generate a `curl` command within {ProjectWebUI} and run it on an unlimited number of hosts to register them using the global registration template.
```

Reply:
I'm against this, because it's not a procedure, I'm describing the methods.

Reply:
OK, then maybe scatch the first word?